crates/nvim-api/src/buffer.rs

```diff
@@ -27,7 +27,7 @@ use crate::LUA_INTERNAL_CALL;
 use crate::{Error, Result};
 
 /// A wrapper around a Neovim buffer handle.
-#[derive(Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
+#[derive(Copy, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
```

Comment:
I'm not against making the handle wrapper types (i.e. `Buffer`, `Window` and `TabPage`) `Copy` (in fact at one point they were), but I'd change them all in the same commit.